### DISCUSSION TITLE
Were do i find the alarm description for all motologix related alarms?

### DISCUSSION THREAD

**Discussion by olsenab** [YaskawaEurope/mlx-examples#28] [Q&A] [answered]
I have seen that i can get alarm description on the pendant, but do you have a complete alarm list with description available?

**Comment by yeu-degroot** [comment, marked as answer]
We recently added a robot alarm section to the docs, see <https://motologix.yaskawa.eu/robot-alarms/>

Tip: You can directly jump to an alarm by using the URL syntax `<alarm number>-<subcode>`

- Example for alarm `4847` with subcode `[2]`: <https://motologix.yaskawa.eu/robot-alarms/yrc1000/4xxx/#4847-2>
